playbooks/robusta_playbooks/restart_loop_reporter.py

```diff
@@ -7,21 +7,22 @@ class RestartLoopParams(RateLimitParams):
     :var restart_reason: Limit restart loops for this specific reason. If omitted, all restart reasons will be included.
     """
 
-    restart_reason: str = None
+    restart_reason: List[str] = None
```

Comment:
This is a breaking change, right?
We need to remember to add it in the release notes

Reply:
Yes, very good point.